s3gof3r_test.go

```diff
@@ -486,5 +485,61 @@ func TestGetterAfterError(t *testing.T) {
 	if err != terr {
 		t.Errorf("expected error %v on Close, got %v", terr, err)
 	}
+}
+
+/* Troubleshooting TUNEBI-64, we noticed multiple files being listed in a single SQS message,
+but the duplicated files were only loaded once.  One bug masking another.  This test verifies
+that GetMultiple isn't responsible for that.  It relies on files currently in the 'tunedb-tetris-beta' S3
+bucket.  To run it you'll need to set your tunedb AWS creds in the usual env vars.  */
```

Comment:
This test probably shouldn't be here since this repo is publicly available (this is a fork of a 3rd party library)